src/cloud/project/routes.md

```diff
@@ -120,11 +145,11 @@ The following sample route does not redirect from the `www` to the apex domain;
 ```yaml
 "http://{default}/":
```

Comment:
Add Example 2: line to label the second example.  

Change explanatory text to 

"In the following example, the route configuration does not redirect URLs from the `www` domain to the apex domain. Instead, requests are served from both the `www` and apex domain."

Reply:
done